.travis.yml

```diff
@@ -66,10 +66,20 @@ before_script:
   - python .misc/check_setuptools.py
 
 script:
-  - py.test
-  - python setup.py bdist_wheel
-  - pip install ./dist/coala-*.whl
-  - pip install coala-bears[alldeps] --pre -U
+  - >
+    if [ ${UNSUPPORTED} == "true"]; then
```

Comment:
I think this should be it's own script. With https://github.com/coala/coala/pull/4704/files#diff-354f30a63fb0907d4ad57269548329e3R72 being a function.